frontend/src/app/testQueue/QueueItem.tsx

```diff
@@ -238,7 +239,11 @@ const QueueItem: any = ({
         .then(testResultsSubmitted)
         .then(refetchQueue)
         .then(() => removeTimer(internalId))
-        .catch((error) => updateMutationError(error));
+        .catch((error) => {
+          updateMutationError(error);
+          // Re-enable Submit in the hopes it will work
+          if (e) e.currentTarget.disabled = false;
```

Comment:
There are a few situations where React might switch DOM elements on us and we'd have to track with a `useRef` but this didn't seem to be one of them (there is no re-render behind the modal dialog) so it should be fine.